### ISSUE TITLE
UTF-8 characters still not showing correctly

### ISSUE THREAD

**Issue by IB-Rahn** [projectsend/projectsend#533]
Hello guys,

so I followed the instructions from the last post of #182 and now my files get saved correctly to the database. At least it displays them correctly in phpMyAdmin. But when I see the file as a client, I still get messed up characters in the title and in the description, nowhere else. The user name, which contains "Umlaute" (e.g. ö, ä) aswell gets displayed correctly. 

What did I do?
- deleted all encode_html functions except the function itself in functions.php
- replaced all `<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />` with `<meta charset="utf-8">`
- convertet the database from utf8mb4_general_ci to utf8_general_ci

Does anyone have a idea what could be wrong?
![phpmyadmin](https://user-images.githubusercontent.com/36037565/37763098-6cb333f4-2dbe-11e8-98e8-8539e5e7dedc.PNG)
![source_code](https://user-images.githubusercontent.com/36037565/37763099-6cce17aa-2dbe-11e8-941f-d397e434e45a.PNG)
![client_view](https://user-images.githubusercontent.com/36037565/37763100-6ce7092c-2dbe-11e8-8bf0-160653d5bc16.PNG)

**Comment by IB-Rahn** [comment]
I really need a solution to this. I can't be the only one. For explanation:

1. Screenshot
This is how the files get saved to the database after I deleted the "encode_html" function

2. Screenshot
That's how the source code looks from "As_Client". I think there is already a mistake with the characters but I don't know where it comes from. 

3. Screenshot
This is how the screen looks for the clients. Not very pretty.

**Comment by mschop** [comment]
Could you please check (and inform us) which charset is used in the database for the schema/table/column. You can find a description how to see this here: https://stackoverflow.com/questions/1049728/how-do-i-see-what-character-set-a-mysql-database-table-column-is

**Comment by mschop** [comment]
Could you please check (and inform us) which charset the schema/table/column has set. See https://stackoverflow.com/questions/1049728/how-do-i-see-what-character-set-a-mysql-database-table-column-is

Note: changing the charset of a schema (database) does not change the charset of tables or columns

**Comment by IB-Rahn** [comment]
Thanks for looking into the problem. Because I already deleted the test system I did another one. Fresh with the newest r1082 and no changes to the files. 
Now the files in the database already get messed up again. 
I checked the charset with the help of the link and they are all in UTF-8

![github_db_charset](https://user-images.githubusercontent.com/36037565/40661067-1918ad1c-6353-11e8-8b2b-38080164900b.PNG)
![github_filename_charset](https://user-images.githubusercontent.com/36037565/40661068-19352c76-6353-11e8-89d8-b7c911af8799.PNG)
![github_tbl_files_charset](https://user-images.githubusercontent.com/36037565/40661069-1963c0f4-6353-11e8-8547-acaa1c3fd892.PNG)

**Comment by mschop** [comment]
Could you please also check the collation? https://stackoverflow.com/questions/3832056/mysql-check-collation-of-a-table

**Comment by IB-Rahn** [comment]
Of course. Here you go.

![github_collation](https://user-images.githubusercontent.com/36037565/40667902-879b4ff0-6363-11e8-8aae-fb8ae29de56a.PNG)

**Comment by mschop** [comment]
> deleted all encode_html functions except the function itself in functions.php

@IB-Rahn did you really remove all calls to encode_html? I would strongly recommend not to do so. Without encode_html your application is insecure against XSS attacks. This is not a solution to the problem.

**Comment by IB-Rahn** [comment]
Yes but I just removed it in the test environment so the security isn't a problem. Is there something else I can try?

**Comment by IB-Rahn** [comment]
Another Update: it seems like the solution doesn't work anymore, when you enable the Editor for the description

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
